src/toast/toast.ts

```diff
@@ -126,10 +122,16 @@ export class NgbToast implements AfterContentInit,
     this.hideOutput.emit();
   }
 
+  private _init() {
+    if (this.autohide && !this._timeoutID) {
+      this._timeoutID = setTimeout(() => this.hide(), this.delay);
+    }
+  }
+
   private _clearTimeout() {
     if (this._timeoutID) {
       clearTimeout(this._timeoutID);
-      this._timeoutID = null;
+      this._timeoutID = undefined;
```

Comment:
It's a question of preference for sure, but I would keep null here, as it's not necessary for the fix. Moreover undefined can be overridden in js, leading to issues, that's I usually prefers null usage.

Reply:
done!